libz-rs-sys/src/gz.rs

```diff
@@ -2269,6 +2269,64 @@ pub unsafe extern "C-unwind" fn gzgets(file: gzFile, buf: *mut c_char, len: c_in
     }
 }
 
+/// Dynamically update the compression level and strategy for `file`. See the
+/// description of [`deflateInit2`] for the meaning of these parameters. Previously
+/// provided data is flushed before applying the parameter changes.
+///
+/// # Returns
+///
+/// - `Z_OK` on success.
```

Comment:
```suggestion
/// - [`Z_OK`] on success.
```

That turns these into clickable links in the docs. Same for the others.